my_artistic_self.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <title>Korey's Art</title>
+        <meta charset="UTF-8">
+        <meta name="description" content="Korey's Artistic Side">
+        <meta name="keywords" content="HTML, CSS, Art, Artistic Side">
+        <meta name="author" content="Korey Goudreau">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>
+            body {
+                background-image:url('pillsburyArt.jpg');
+                background-repeat:no-repeat;
+                background-attachment:fixed;
+                background-size:contain;
+                background-position:center;
+                background-color:maroon;
+                color:aliceblue;
+                text-shadow: 2px 2px black;
+            }
+            a {
+                background-color:white;
+                text-shadow:none;
+                border-radius:5px;
+                padding:5% 5%;
+                font-size:50px;
+            }
+            h1, div {
+                text-align:center;
+                margin:auto;
+                width:50%;
+                padding:50px;
+            }
+            h2 {
+                text-align:center;
+                margin:auto;
+                width:50%;
+                color:magenta;
+                background-color:pink;
+            }
+            p {
+                font-family:"Lucida Console", "Courier New", monospace;
+            }
+            #class1 {
+                font-size:xx-large;
+                color:rgb(0, 248, 54);
+                position:fixed;
+                bottom:105px;
+                left:40%;
+            }
+            #class2 {
+                font-size:55px;
+                color:darkgoldenrod;
+                position:relative;
+                transform: rotate(15deg);
+                left:20%;
+                width:fit-content;
+                height:fit-content;
+            }
+            #class3{
+                font-size:200%;
+                color:rgb(245, 54, 239);
+                position:sticky;
+                text-orientation:upright;
+                writing-mode:vertical-lr;
+                top:10%;
+                left:90%;
+            }
+            #class4 {
+                font-size:75px;
+                color:gold;
+                position:static;
+                width:fit-content;
+                height:fit-content;
+                transform:skewY(-20deg);
+            }
+            #class5 {
+                font-size:4em;
+                color:rgb(61, 33, 244);
+                position:relative;
+                bottom:275px;
+                left:60%;
+                width:fit-content;
+                height:fit-content;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>This corner is for Self Expression!</h1>
+        <p id="class1">Hardworking</p>
```

Comment:
It's a little confusing to have name your ids as "classx"! The id is *not* a class! It is an identity name. The class would be given with `<p id="keyword1" class="keywords">`. You still managed to give a common font to all keywords by modifying the font to all paragraphs `p`, but that's not exactly what was asked.